web/frontend/src/components/slam-2d-render.vue

```diff
@@ -179,19 +363,31 @@ onMounted(() => {
   run();
 
   if (props.pointcloud !== undefined) {
-    updateCloud(props.pointcloud);
+    updatePointCloud(props.pointcloud);
   }
 
   if (props.pose !== undefined) {
     updatePose(props.pose);
   }
+
 });
 
 onUnmounted(() => {
   pause();
   disposeScene();
 });
 
+watch(() => [props.destVector!.y, props.destVector!.x, props.destExists], updateOrRemoveDestinationMarker);
+
+watch(() => props.axesVisible, () => {
+  for (const gridPart of gridSubparts) {
+    const part = scene.getObjectByName(gridPart);
```

Comment:
[q] Could 'part' here be renamed to axis? Part seems a bit generic if this only stores axes

Reply:
hmmm, 'axis' would not be the right name for this since gridSubparts contains the grey background grid as well as the axes, hence the generic name.